src/Sample.java

```diff
@@ -1,6 +1,6 @@
 class Sample {
     int ctz0(int value) {
-        if (0 <= value) {
+        if (0 < value) {
             value--;
         }
         if (value < 0) {
```

Comment:
In this Pull Request, this code fragment is modified,
but the following unmodified.
https://github.com/T45K/trial/blob/3e1c4980464ba3e121b1ef765244af045df01735/src/Sample.java#L12-L20

Why don't you modify consistently?